pkg/report/testdata/starnix/report/0

```diff
@@ -0,0 +1,84 @@
+TITLE: starnix kernel panic: explicit panic in src/starnix/kernel/runner/container.rs: failed to start container.: errno NUM, d
```

Comment:
Nit: does it always end with `errno X`? If yes, maybe we could adjust the regexp to only cut the part before it / right after it, so that there's no `, d`?

Reply:
In general, the message won't contain `errno X, details: ...` -- it's specific to this panic, and other panics will include their own message here with no special format. For now, I was thinking that it's enough to just print the beginning of the panic message until the title hits the character limit. (In the most recent push, I've changed "explicit panic" to "panic" since it's both more accurate and leaves more space for the detailed message.)

Reply:
I see, thanks for the explanation.

From such titles we expect that they be as similar between different crashes caused by the same bug as possible -- otherwise you end up having lots of bug duplicates (it's bad on a local syz-manager instance and much much worse if it's deployed on syzbot). So even if the title is truncated, but in a consistent way, there should be no problems because of that.

Reply:
Sounds good. So far, it seems like this format is working well to avoid duplicate titles, even if it's a bit ugly to have some messages truncated in the middle of a word.